packages/@lightningjs/ui-components/src/components/Artwork/Artwork.d.ts

```diff
@@ -18,11 +18,11 @@
 
 import type lng from '@lightningjs/core';
 import type Base from '../Base';
```

Comment:
I don't know how the `type` keyword got into all of these but we can(should) remove it from import lines. it's used by TS to ensure imports used just for types dont get compiled into the final JS files - it isn't necessary in declaration files